mautic/mautic
# switch to md5 for password hashing during tests to improve speed

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

By default, Mautic uses `bcrypt` for password hashing.
While this makes sense for production environments, it has no real use during tests.
It's a "slow" hasher compared to e.g. `md5`

This PR switches the algorithm to md5, which results in faster tests.

**before**:
<img width="325" alt="Screenshot 2022-09-23 at 11 44 44" src="https://user-images.githubusercontent.com/3983285/191934492-edbb5db7-8175-4954-b3ec-abfed511cf5b.png">

**after**:
<img width="286" alt="Screenshot 2022-09-23 at 12 29 16" src="https://user-images.githubusercontent.com/3983285/191942102-c47cbdbc-b820-452c-a972-853e8f4b1c13.png">




#### Steps to test this PR:


1. Check the outcome of the tests, and compare the time needed to complete the PHPUnit tests with a recent test run without this fix (e.g. https://github.com/mautic/mautic/actions/runs/3111657640)
